core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -123,6 +124,7 @@ def __init__(self,
 
         self.fit_time = None  # Time taken to fit in seconds (Training data)
         self.predict_time = None  # Time taken to predict in seconds (Validation data)
+        self.predict_1_time = None  # Time taken to predict 1 row of data in seconds (with batch size `predict_1_batch_size` in params_aux)
```

Comment:
numbers in variable names are ok I think, @gradientsky feel free to comment. I generally try to avoid overly long variable names when they don't add much to the clarity.

For example, if we did this then we would also have:

```
predict_one_row_batch_size
```

which is starting to get pretty long